### DISCUSSION TITLE
ios18 introducing new algorithm for sync'ing audio?

### DISCUSSION THREAD

**Discussion by dltacube** [mikebrady/shairport-sync#1905] [General]
I've got some devices on ios18 beta and noticing they're not properly sync'd with ios17 devices OR shairplay devices. None of the usual things seem to resolve this (hitting pause then play, restarting altogether, etc).

Has anyone else noticed this behavior or is it just me?

/edit I am on 4.3.1 so let me update and report back

**Comment by mikebrady** [comment]
Thanks. I haven’t noticed this, TBH…

**Reply by dltacube** [reply]
ios 18 is in beta right now. Are any of your speakers on it yet?

**Reply by dltacube** [reply]
I tried to reset my homepods to see if I could get them on ios17 but it just put them back on 18 =\

**Reply by dltacube** [reply]
I'm also a few versions behind so let me update and report back

**Comment by dltacube** [comment]
I'm still having issues between just homepods on different versions...not sure what's going on maybe it's my network being unstable.

**Reply by mikebrady** [reply]
Thanks. Unfortunately I am away from my only HomePod mini for another three weeks, so can’t offer any insight just now. But during the betas, sync between the HPm and Shairport Sync was (unremarkably) fine.

**Reply by dltacube** [reply]
No worries. If this isn't a "me" problem you'll get more reports since 18 went live yesterday.

**Comment by ghost** [comment]
I see nothing wrong playing to shairport and my mac.
i18 / 4.3.4-1-g910264e5-AirPlay2-smi10-alac-OpenSSL-Avahi-ALSA-pa-pw-stdout-pipe-soxr-convolution-metadata-dbus-mpris-sysconfdir:/etc
